wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -9,35 +9,63 @@ tags:
+
+***Key note:*** ***A member of the NAT can engage with any task within either subgroup despite what their subgroup may be listed as.*** For example, if a member of the NAT is an evaluator, they may still make announcements or participate in discussions. Assigned subgroups only describe what each member is formally responsible for.
+
+## Activity
+
+Depending on their subgroup, each NAT member has different activity requirements. Evaluators are expected to consistently evaluate both applicants and current BNs, while also keeping up to date with the mapping/modding scene on their own through modding. Advisors are expected to uphold key parts of the ranking process on a case-by-case basis.
+
+Members who are found to be inactive will be confronted by other members of the NAT. If a timely resolution to their inactivity isn't feasible, they will be removed from the NAT. Members of the Evaluator subgroup working on other mapping-related projects may be moved to the Advisor subgroup to better reflect their contribution.
 
 ## Promotion to the NAT
 
-Before joining the NAT, a user must either be a full member of the Beatmap Nominators, or previously an NAT member and still involved in the community. Most NAT candidates are initially considered based on their commitment to helping the mapping and modding community, and further demonstration of their ability to contribute to all four subcategories of NAT responsibilities is usually the basis for being promoted.
+Before joining the NAT, a user must either be a full member of the Beatmap Nominators, or previously an NAT member and still involved in the community. Most NAT candidates are initially considered based on their commitment to helping the mapping and modding community, and further demonstration of their ability to contribute to a multitude of NAT responsibilities is usually the basis for being promoted.
 
-Since the majority of new NAT members will help with BN evaluation, it's important that NAT candidates are very proficient in assessing the proficiency of others. Full BNs have the option to do mock evaluations alongside the NAT's real ones, giving them an opportunity to practice. NAT candidates have a much better chance of being promoted if their evaluations are thorough and come to similar conclusions as the NAT (or have supportive reasoning otherwise).
+Since all new NAT members start off as evaluators, it's important that NAT candidates are exceptional in assessing the proficiency of others. Full BNs may be given the opportunity to participate in mock evaluations or temporarily join the evaluation team, giving them an opportunity to practice. NAT candidates have a much better chance of being promoted if their evaluations are thorough and come to similar conclusions as the NAT (or have supportive reasoning otherwise).
```

Comment:
```suggestion
Since all new NAT members start off as evaluators, it is important that NAT candidates are exceptional in assessing the proficiency of others. Full BNs may be given the opportunity to participate in [mock evaluations](/wiki/People/The_Team/Nomination_Assessment_Team/Evaluations#mock-evaluations) or temporarily join the evaluation team, giving them an opportunity to practice. NAT candidates have a much better chance of being promoted if their evaluations are thorough and come to similar conclusions as the NAT (or have supportive reasoning otherwise).
```